src/cfnlint/data/DownloadsMetadata/d3f122354444e0d1662c8fb30a66be635537862cfdcc63e1cedb9d28c61e5872.meta.json

```diff
@@ -0,0 +1 @@
+{"url": "https://docs.oasis-open.org/sarif/sarif/v2.1.0/cos02/schemas/sarif-schema-2.1.0.json", "etag": "\"5a0aa71f02000\""}
```

Comment:
You can delete this file.  This was a file created by a test on mistake.

Reply:
Will remove

Reply:
Removed